### PR TITLE
[Fix] fix cobbler_system ssl cert checkwhen validate_certs set to no

### LINKED ISSUE #1878: Cobbler module cobbler_system validate_certs error
**Summary**

When using the `cobbler_system` with `validate_certs` flag set to `no`.   The module is validating certificate:
```
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Connection to 'https://xxxx.domain:443/cobbler_api' failed. [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1123)"}
```

The bug was present in the main project. See [here](https://github.com/ansible/ansible/issues/52239)

**Issue Type**

Bug Report

**Component Name**

cobbler_system

**Ansible Version**

```paste below
ansible-playbook 2.9.14
  config file = /home/nico/Documents/tower-automation/ansible.cfg
  configured module search path = ['/home/nico/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/nico/.local/pipx/venvs/ansible/lib/python3.8/site-packages/ansible
  executable location = /home/nico/.local/bin/ansible-playbook
  python version = 3.8.5 (default, Jul 28 2020, 12:59:40) [GCC 9.3.0]
Using /home/nico/Documents/tower-automation/ansible.cfg as config file

```

**Configuration**

```paste below
default conf. No changes
```

**OS / Environment**

Ubuntu 20.04

**Steps To Reproduce**

When using the `cobbler_system` with `validate_certs` flag set to `no`
```yml
    - name: Create a system
      cobbler_system:
        host: terre.gre.hpecorp.net
        validate_certs: no
        use_ssl: true
        username: me
        password: mypass
        name: siam014.bck
        properties:
          hostname: siam014.gre.hpecorp.net
          profile: HOE-CENTOS8.3-SPP2009-x86_64-NOPARTNUMBER-RC1
        interfaces:
          eth0:
            macaddress: 00:50:56:a6:3a:33
            ipaddress: 16.17.88.14
            dhcptag: cms_vlan152
            dnsname: siam014.gre.hpecorp.net
        sync: false
      delegate_to: localhost
```

**Expected Results**

System created without any error

**Actual Results**

The ssl cert check is not skipped.
```paste below
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Connection to 'https://terre.gre.hpecorp.net:443/cobbler_api' failed. [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1123)"}
```

### PR DESCRIPTION
**Pull request by Sispheor**
##### SUMMARY
Fixes  #1878.
cobbler_system now check self signed certificates correctly. Followed implementation [pep-0476](https://www.python.org/dev/peps/pep-0476/).

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
cobbler_system

### CODE REVIEW

**Comment by felixfontein** [comment]
This part of Python is really a mess... The change looks reasonable to me.

Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to)?

**Comment by Sispheor** [comment]
Done

**Comment by Sispheor** [comment]
I've added the same part of code into the other cobbler module "cobbler_sync"

File: changelogs/fragments/1880-fix_cobbler_system_ssl.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - fix cobbler_system ssl cert checkwhen validate_certs set to no
```
Comment (felixfontein) [inline changelogs/fragments/1880-fix_cobbler_system_ssl.yml:2]:
```suggestion
  - cobbler_sync, cobbler_system - fix SSL/TLS certificate check when ``validate_certs`` set to ``false`` (https://github.com/ansible-collections/community.general/pull/1880).
```

Comment (Sispheor) [reply changelogs/fragments/1880-fix_cobbler_system_ssl.yml]: Renamed and squashed into a single commit.

**Review by felixfontein — APPROVED** [review]

File: changelogs/fragments/1880-fix_cobbler_system_ssl.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  -  cobbler_sync, cobbler_system - fix SSL/TLS certificate check when ``validate_certs`` set to ``false`` (https://github.com/ansible-collections/community.general/pull/1880).
```
Comment (felixfontein) [inline changelogs/fragments/1880-fix_cobbler_system_ssl.yml:2]: To make the linter happy:
```suggestion
  - cobbler_sync, cobbler_system - fix SSL/TLS certificate check when ``validate_certs`` set to ``false`` (https://github.com/ansible-collections/community.general/pull/1880).
```

Comment (Sispheor) [reply changelogs/fragments/1880-fix_cobbler_system_ssl.yml]: Updated

**Review by felixfontein — APPROVED** [review]
If nobody complains, I'll merge this later today or tomorrow.

**Comment by felixfontein** [comment]
Now it's later, so merged ;-)

@Sispheor thanks for fixing this!

**Pull request merged**